docs/antrea-agent-simulator.md

```diff
@@ -0,0 +1,48 @@
+# Run Antrea agent simulator
+
+## 1. build the simulator image
+```bash
+make build-scale-simulator
+```
+
+## 2. build the kubemark image
+```bash
+cd $KUBERNETES_PATH
+make WHAT=cmd/kubemark KUBE_BUILD_PLATFORMS=linux/amd64
+cp ./_output/local/go/bin/linux_amd64/kubemark cluster/images/kubemark
+```
+change the image in Dockerfile to other base images if pull failed, for example(antrea/base-ubuntu:2.14.0)
```

Comment:
I think we don't need to mention this in doc specically. And actually I can pull the image.

Reply:
I put it in vmware's private image repository, shall I also push it to antrea?